lightning/src/routing/router.rs

```diff
@@ -990,7 +1005,7 @@ where L::Target: Logger {
 	// meaning how much will be paid in fees after this node (to the best of our knowledge).
 	// This data can later be helpful to optimize routing (pay lower fees).
 	macro_rules! add_entries_to_cheapest_to_target_node {
-		( $node: expr, $node_id: expr, $fee_to_target_msat: expr, $next_hops_value_contribution: expr, $next_hops_path_htlc_minimum_msat: expr, $next_hops_path_penalty_msat: expr ) => {
+		( $node: expr, $node_id: expr, $fee_to_target_msat: expr, $max_total_cltv_expiry_delta: expr, $next_hops_value_contribution: expr, $next_hops_path_htlc_minimum_msat: expr, $next_hops_path_penalty_msat: expr, $next_hops_cltv_delta: expr ) => {
```

Comment:
Likewise here.

Reply:
Addressed in https://github.com/lightningdevkit/rust-lightning/pull/1234/commits/cb7269de6a7ffa38aa7cdd222ab699c97e6eac8a.